styles/find-and-replace.less

```diff
@@ -166,6 +166,25 @@ atom-workspace.find-visible {
       margin-right: @component-padding;
     }
   }
+
+  .wrap-icon {
+    @wrap-size: 96px;
+
+    display: none;
+    position: absolute;
+    left: 50%;
+    top: @wrap-size * -1.5;
```

Comment:
Any reason why this is toward the bottom of the window? I'd prefer it in the center of the window.